stock_customer_deposit/__manifest__.py

```diff
@@ -10,7 +10,7 @@
     "website": "https://github.com/OCA/stock-logistics-workflow",
     "author": "Moduon, Odoo Community Association (OCA)",
     "maintainers": ["rafaelbn", "EmilioPascual"],
-    "license": "AGPL-3",
+    "license": "LGPL-3",
```

Comment:
Sorry @EmilioPascual, you amended the wrong commit (the one adding the rebel configuration, not the one creating the module). Could you please fix the amend to merge?